src/main/java/net/sf/jabref/gui/maintable/MainTableSelectionListener.java

```diff
@@ -183,11 +183,10 @@ public void editSignalled() {
 
     public void editSignalled(BibEntry entry) {
         final BasePanelMode mode = panel.getMode();
-        EntryEditor editor = panel.getEntryEditor(entry);
```

Comment:
The new editor was only needed if the condition was met, and previously the focus was possibly set to the wrong editor (if the condition wasn't met).